Uzdavinys07/Program.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+
+namespace Uzdavinys07
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+
+
+            // Vardo ir amžiaus įvedimas
+            Console.Write("Įveskite savo vardą ir amžių: ");
+            string vardas = Console.ReadLine();
+            var amzius = Console.ReadLine();
```

Comment:
Rekomendacija nenaudoti `var`. Be to eilutėje aukščiau kūrėte `string vardas`, tai čia taip pat galėtų būti `string amzius`. Kita rekomendacija yra, kad jeigu jau turite kintamąjį kuris pagal pavadinimą lyg ir pasako koks tai duomenų tipas, tuomet toks tipas ir turėtų būti. Kadangi kintamasis `amzius` iš tikro yra `string` tipo, tai tinkamesnis pavadinimas būtų pavyzdžiui `amziusString`, arba dar geriau būtų konvertuoti `string` į `int` reikšmę.